packages/attendance/src/components/AttendanceComponent.js

```diff
@@ -939,7 +940,7 @@ const CalendarComponent = ({
               {!isIconSizeSmall ? (
                 <VStack alignItems={"center"}>
                   {index === 0 ? (
-                    <Text pb="1" color={"attendanceCardText.400"}>
+                    <Text pb="1" fontSize='12px' fontFamily="Inter" color={"attendanceCardText.400"}>
```

Comment:
We started creating h1, h2, h3... we should use those - if this doesn't exist, we should create one.